diff --git a/sugarcrm/include/utils/activity_utils.php b/sugarcrm/include/utils/activity_utils.php
index 8fdfc31..dfce4f1 100755
--- a/sugarcrm/include/utils/activity_utils.php
+++ b/sugarcrm/include/utils/activity_utils.php
@@ -37,15 +37,21 @@ if(!defined('sugarEntry') || !sugarEntry) die('Not A Valid Entry Point');
 
 function build_related_list_by_user_id($bean, $user_id,$where) {
 	$bean_id_name = strtolower($bean->object_name).'_id';
-
-	$select = "SELECT {$bean->table_name}.* from {$bean->rel_users_table},{$bean->table_name} ";
-
+    $select = '';
 	$auto_where = ' WHERE ';
 	if(!empty($where)) {
 		$auto_where .= $where. ' AND ';
 	}
+    
 
-	$auto_where .= " {$bean->rel_users_table}.{$bean_id_name}={$bean->table_name}.id AND {$bean->rel_users_table}.user_id='{$user_id}' AND {$bean->table_name}.deleted=0 AND {$bean->rel_users_table}.deleted=0";
+    if($bean->table_name == "meetings") {
+	    $select = "SELECT {$bean->table_name}.* from {$bean->table_name} ";
+    	$auto_where .= " {$bean->table_name}.deleted=0 ";
+    }
+    else {
+	    $select = "SELECT {$bean->table_name}.* from {$bean->rel_users_table},{$bean->table_name} ";
+        $auto_where .= " {$bean->rel_users_table}.{$bean_id_name}={$bean->table_name}.id AND {$bean->rel_users_table}.user_id='{$user_id}' AND {$bean->table_name}.deleted=0 AND {$bean->rel_users_table}.deleted=0";
+    }
 
 
 
diff --git a/sugarcrm/modules/Meetings/metadata/additionalDetails.php b/sugarcrm/modules/Meetings/metadata/additionalDetails.php
index 152dce5..a41b481 100755
--- a/sugarcrm/modules/Meetings/metadata/additionalDetails.php
+++ b/sugarcrm/modules/Meetings/metadata/additionalDetails.php
@@ -45,6 +45,9 @@ function additionalDetailsMeeting($fields) {
 	$overlib_string = '';
 	
 	//Modify by jchi 6/27/2008 1515pm china time , bug 20626.
+   // Show assigned user name in the popup (to know whose event it is)
+	if(!empty($fields['ASSIGNED_USER_ID'])) 
+		$overlib_string .= '<b>'. $mod_strings['LBL_ASSIGNED_TO_NAME'] . '</b> ' . $fields['ASSIGNED_USER_NAME'] . ' <br>';
+
 	if(!empty($fields['DATE_START'])) 
 		$overlib_string .= '<b>'. $mod_strings['LBL_DATE_TIME'] . '</b> ' . $fields['DATE_START'] . ' <br>';
 	if(isset($fields['DURATION_HOURS']) || isset($fields['DURATION_MINUTES'])) {
